docs/apis/autopeering.md

```diff
@@ -0,0 +1,120 @@
+# Peering API Methods
+
+Peering API allows retrieving basic information about autopeering.
+
+The API provides the following functions and endpoints:
+
+* [/autopeering/neighbors](#autopeeringneighbors)
+
+
+Client lib APIs:
+* [GetAutopeeringNeighbors()](#client-lib---getautopeeringneighbors)
+
+
+
+##  `/autopeering/neighbors`
+
+Returns the chosen and accepted neighbors of the node.
+
+
+### Parameters
+
+| **Parameter**            | `known`      |
+|--------------------------|----------------|
+| **Required or Optional** | optional       |
+| **Description**          | Return all known peers, set to `1` (default: `0`)   |
+| **Type**                 | int         |
```

Comment:
The table is horizontal. Vertical tables are more common and is what you've used in the other tables. Can we make it consistent?

Reply:
I kinda like this layout for parameters tbh, which you can really focus on details of each of them.
Plus, we did the same for other files, if we changed it here, the same should go to other files. And that will take us a lot of time to do so. :sweat_smile: